pom.xml

```diff
@@ -116,7 +116,7 @@
             <dependency>
                 <groupId>org.postgresql</groupId>
                 <artifactId>postgresql</artifactId>
-                <version>42.3.1</version>
```

Comment:
Should hopefully fix https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21724